lib/jekyll/utils.rb

```diff
@@ -185,6 +188,9 @@ def has_yaml_header?(file)
     #   slugify("The _config.yml file", "ascii")
     #   # => "the-config.yml-file"
     #
+    #   slugify("The _config.yml file", "latin")
```

Comment:
I think it would be better to see the example here demonstrate the difference between the ascii mode and the latin mode. This will make it clearer as to why the latin mode exists.

Reply:
@mattr- Yeah, I thought so, too. To really demonstrate the difference, the input string would need an accented character, but when I tried that, one of the builds failed with a note about non-ASCII characters disallowed in comments. I would point out the failed build but I'm not sure how to go through the build history for this PR in TravisCI.